LPS-143393 | master

ozysouza opened the pull request:
https://issues.liferay.com/browse/LPS-143393

cc @david-gutierrez-mesa 
cc @Tim-Cao

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/macros/ControlMenu.macro:
```diff
@@ -1,5 +1,11 @@
 definition {
 
+	macro toggleControls {
```
Rename to switchToggleControls then move to WidgetPages.macro

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/macros/ControlMenu.macro:
```diff
@@ -13,4 +19,10 @@ definition {
 			value1 = "${headerTitleLabel}");
 	}
 
+	macro viewToggleControls {
```
move to WidgetPages.macro

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:
```diff
@@ -41,15 +45,15 @@ definition {
 		property portal.acceptance = "true";
 
-		task ("Can hide and show existing invisible portlets on widget page via switch toggle controls.") {
+		task ("View toggle controls is at view action") {
```
View toggle controls is at view status

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:
```diff
@@ -77,10 +79,10 @@ definition {
-			Click(
-				key_status = "hidden",
-				locator1 = "ControlMenu#TOGGLE_CONTROLS");
+		task ("Switch Toggle Controls to unhidden then view portlets are shown") {
```
task ("Switch Toggle Controls to view then invisible widgets are shown") {

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:
```diff
@@ -41,15 +45,15 @@ definition {
+			ControlMenu.viewToggleControls(status = "view");
+		}
 
+		task ("Add a Web Content Display widget to page then view portlets are shown") {
```
View invisible widgets are shown on page

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:
```diff
@@ -61,14 +65,12 @@ definition {
-			Click(
-				key_status = "view",
-				locator1 = "ControlMenu#TOGGLE_CONTROLS");
+		task ("Switch Toggle Controls to hidden then view portlets are not shown") {
```
task ("Switch Toggle Controls to hidden then invisible widgets are not shown") {

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:
```diff
@@ -111,15 +109,15 @@ definition {
 		property portal.acceptance = "true";
 
-		task ("Switch Toggle Controls to hidden") {
+		task ("View toggle controls is at view action") {
```
task ("View toggle controls is at view status") {

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:
```diff
@@ -226,32 +200,26 @@ definition {
 	@description = "This is a use case for LPS-108216. Switch toggle controls shouldn't affect portlets on content page."
 	@priority = "4"
-	test ToggleControlsNotAffectPortletsOnContentPage {
+	test NotAffectPortletsOnContentPage {
```
ViewNotAffectWidgetsOnContentPage

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:
```diff
@@ -287,8 +255,8 @@ definition {
 	@priority = "4"
-	test ToggleControlsOnlyAffectCurrentInstance {
-		property test.name.skip.portal.instance = "ToggleControls#ToggleControlsOnlyAffectCurrentInstance";
+	test OnlyAffectCurrentInstance {
```
ViewOnlyAffectCurrentInstance

Tim-Cao reviewed (commented):
Hi @ozysouza, could you update according to comments? Thanks

ozysouza commented:
> Hi @ozysouza, could you update according to comments? Thanks

Hi @Tim-Cao, PR updated following your comments. I removed the LocalizationWithSitePages because it was done in another LPS and I added here my mistake.

Tim-Cao commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-source-format/959/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/959/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/758c99bd5bac7693e03bf1ec877eaf329bd6f229">758c99bd5bac7693e03bf1ec877eaf329bd6f229</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/ozysouza/liferay-portal/tree/LPS-143393">LPS-143393</a><br/>Branch GIT ID: <a href="https://github.com/ozysouza/liferay-portal/commit/2cd29c284be25e8e512dbff815e7b98870ccef2c">2cd29c284be25e8e512dbff815e7b98870ccef2c</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-source-format/959/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/959/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/959/">test-portal-source-format#959</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/959/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/6602">liferay-echo#6602</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1588038638">[master] ci:test:sf - ozysouza &gt; liferay-echo - PR#6602 - 2021-12-06[07:25:49]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/3076/">publish-testray-report#3076</a><br/></div></div>

Tim-Cao commented:
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/2cd29c284be25e8e512dbff815e7b98870ccef2c...0e17fa39aba0ea21504078a918d1c87f8944dcbf

Tim-Cao commented:
ci:close
